Sources/ApolloCodegenLib/Templates/TemplateRenderer.swift

```diff
@@ -111,6 +111,14 @@ extension TemplateRenderer {
     """
     ).description
   }
+
+  func embeddedAccessControlModifier(
```

Comment:
Yes it would be a good change to get it into `TemplateRenderer` as you mentioned. It started out that not many templates needed to reference the configuration but it's way more complicated now.